src/Input.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+const onChange = jest.fn();
+const onClick = jest.fn();
+
+const testValue = '123';
+
+const renderInputTemplate = (value = '') => render((
+  <Input value={value} onChange={onChange} onClick={onClick} />
```

Comment:
```suggestion
  <Input 
    value={value}
    onChange={onChange}
    onClick={onClick}
   />
```

저는 `props` 가 2개 이상일 때는 줄바꿈 해줘서 코드 가독성을 높이려고 하는 편인데 비교해봤을 때 어떠하신가요?